src/provider/doc/ibmca-provider.man

```diff
@@ -96,11 +96,23 @@ provider if you are on an IBM z15 or later. This would actually make it slower.
 Enables debug output for the IBMCA provider. If this option is not specified,
 no debuging output is produced. If \fBdebug = stderr\fP is specified,
```

Comment:
This is old text, but you may want to correct the typo while at it: 
debuging -> debugging.

Reply:
fixed